src/components/AppNavbar.tsx

```diff
@@ -136,7 +136,7 @@ const AppNavbar = () => {
 								</DropdownMenuLinkDescription>
 							</DropdownMenuLink>
 							{import.meta.env.VITE_SHOW_STORYBOOK === 'true' && (
-								<DropdownMenuLink to="/storybook" reloadDocument>
+								<DropdownMenuLink to="/storybook/" reloadDocument>
```

Comment:
Was this meant to be included in this PR?

Reply:
@hminsky2002 when reviewing I recommend going "commit by commit" -- you can see the rationale for this change in the commit message that included it: https://github.com/PhilanthropyDataCommons/front-end/pull/852/commits/a04c1b34e02995e7ef107879b353f29b103f7b69